sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -412,7 +418,7 @@ class DatasetSuite extends QueryTest with SharedSQLContext {
 }
 
 
-case class ClassData(a: String, b: Int)
+case class ClassData(a: String, b: Integer)
```

Comment:
some other tests may depend on the fact that `ClassData.b` is not nullable, we'd better not break it.